### ISSUE TITLE
Export ask password interactively

### ISSUE THREAD

**Issue by somzoli** [EvilFreelancer/routeros-api-php#119]
Hi!

If i use code example below, the export query ask password interactively instead using client parameters.
Can you help with this? 

```
use \RouterOS\Client;

// Initiate client with config object
$client = new Client([
    'host'        => '192.168.1.3',
    'user'        => 'admin',
    'pass'        => 'admin',
    'ssh_port'    => 22222,
    'ssh_timeout' => 60, // if not set then 30 seconds by default 
]);

// Execute export command via ssh
$response = $client->query('/export');
// or
$response = $client->export();

var_dump($response);
```

Thanks!

**Comment by EvilFreelancer** [comment]
Hi! Export feature in this library is based on ssh extension, and i guess ssh daemon is not preconfigured on your router.

Can you manually connect via ssh to device with the same username and password?

**Comment by somzoli** [comment]
Yes, i can able to connect via ssh successfully with same username/password. 
When it try to connect through php the login prompt shows this: admin+etc@192.168.X.X's password:
After give the correct password, the dump work as expected. 
Seems like the 'pass' not used in the library.

**Comment by somzoli** [comment]
With 'ssh_private_key' option the connection work, but i need password auth if it possible.

**Comment by EvilFreelancer** [comment]
Sounds like something changed in https://github.com/spatie/ssh

I'll check it later, thanks!

**Comment by EvilFreelancer** [comment]
Hi! PR #120 merged, fix available in version `1.5.3`.
